.github/CONTRIBUTING.md

```diff
@@ -31,27 +34,33 @@ If you're developing against Magic Modules, vendoring changes is done automatica
 ### Running Tests
 
 The following environment variables must be set in order to run tests:
+
 ```
 GOOGLE_PROJECT
-GOOGLE_CREDENTIALS or GOOGLE_USE_DEFAULT_CREDENTIALS
+GOOGLE_CREDENTIALS
+GOOGLE_REGION
+GOOGLE_ZONE
 ```
-For certain tests, the following variables may also need to be set:
+
+To ensure that your tests are performed in a region and zone with wide support for GCP feature, `GOOGLE_REGION` should be set to `us-central1` and `GOOGLE_ZONE` to `us-central1-a`.
+
+For certain tests, primarily those involving project creation, the following variables may also need to be set. Most tests do
+not require their being set:
+
 ```
-GOOGLE_REGION
 GOOGLE_ORG
 GOOGLE_BILLING_ACCOUNT
 ```
 
-The only region we support running tests in right now is `us-central1` - some products that are tested here are only available in a few regions, and the only region that all products are available in is `us-central1`.
+To run a specific test, use `TESTARGS`, such as in:
 
-To run a specific test, use a command such as:
 ```
 make testacc TEST=./google TESTARGS='-run=TestAccContainerNodePool_basic'
 ```
 
 The `TESTARGS` variable is regexp-like, so multiple tests can be run in parallel by specifying a common substring of those tests (for example, `TestAccContainerNodePool` to run all node pool tests).
 
-To run all tests, you can simply omit the `TESTARGS` argument - but please keep in mind that that is quite a few tests and will take quite a long time and create some fairly expensive resources.  It usually is not advisable to run all tests.
+To run all tests, you can omit the `TESTARGS` argument - but please keep in mind that that is quite a few tests and will take quite a long time and create some fairly expensive resources.  It usually is not advisable to run all tests.
```

Comment:
I'd make this paragraph a little scarier and maybe bold 'fairly expensive resources.'

The tests spin up so much stuff and it can cost a lot of money.

Reply:
I think this is fine because every example includes a `TESTARGS` and anyone reading this will pick up on the fact that it will run _every_ test (and they take hours to run so you'd notice pretty quickly that you did something Not Good)